web/settings.py

```
@@ -386,3 +387,10 @@
 
 # GitHub API Token for fetching contributor data
 GITHUB_TOKEN = os.environ.get("GITHUB_TOKEN", "")
+
+# Channel Layers Configuration
+CHANNEL_LAYERS = {
+    "default": {
+        "BACKEND": "channels.layers.InMemoryChannelLayer"
```

Ruff (0.11.9)

Trailing comma missing

Add trailing comma

(COM812)